pallets/vesting-manager/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+use frame_support::traits::VestingSchedule;
+use sp_runtime::traits::StaticLookup;
+
+type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(_);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+		type VestingSchedule: VestingSchedule<Self::AccountId>;
+	}
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		VestingScheduleRemoved { who: T::AccountId, schedule_index: u32 },
+	}
+
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		#[pallet::call_index(0)]
+		#[pallet::weight(10_000_000)]
```

Comment:
I didn't bother to benchmark the actual weight. I just used a high number and then made the transaction feeless if really called from root.

Shall we benchmark it or not worth the time?

Reply:
I don't think it's worth the time and we should not add benchmarks for this pallet.